### ISSUE TITLE
Failed to resolve

### ISSUE THREAD

**Issue by TolikB** [Salat-Cx65/AdvancedBiometricPromptCompat#1]
Failed to resolve: com.microsoft.device.dualscreen:fragmentshandler:1.0.0-alpha02
Failed to resolve: com.tencent.soter:soter-core:+
Failed to resolve: com.microsoft.device.dualscreen:core:1.0.0-alpha01

**Comment by Salat-Cx65** [comment]
Hi.
Please check https://github.com/Salat-Cx65/AdvancedBiometricPromptCompat#setup - you need to add some extra repositories to your Gradle:

```scala
 repositories {  
 maven { url 'https://developer.huawei.com/repo/' }  
 maven { url "https://dl.bintray.com/tencent-soter/maven/" }  
 maven { url "https://pkgs.dev.azure.com/MicrosoftDeviceSDK/DuoSDK-Public/_packaging/Duo-SDK-Feed/maven/v1" }  
 } 
 ```
 
I'm looking for a way how to manage this problem use the POM files to avoid this extra step and simplify the setup.

**Comment by TolikB** [comment]
i added all this  but  get same result and then write issues)

**Comment by Salat-Cx65** [comment]
Fixed in 1.0.4

**Comment by Salat-Cx65** [comment]
Reopened cause of some issues with JCenter happens
